recipes/meme/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+MEME_ETC_DIR=${PREFIX}/etc
+cpanm YAML
+cpanm HTML::PullParser
```

Comment:
Thanks @bgruening, but it leads to many recursive perl dependencies some of them failing tests.